src/core/jpg.js

```diff
@@ -332,6 +332,7 @@ var JpegImage = (function JpegImageClosure() {
     if (!resetInterval) {
       resetInterval = mcuExpected;
     }
+    var nonDefaultResetInterval = resetInterval !== mcuExpected;
```

Comment:
I don't think so, and quick testing seem confirm it, since provided that I'm reading the specification correctly (see the quotes above) it should only be the *last* `mcu` where the `resetInterval` could potentially be an issue.